torchgeo/datamodules/chesapeake.py

```diff
@@ -94,7 +91,7 @@ def __init__(
         # This is a rough estimate of how large of a patch we will need to sample in
         # EPSG:3857 in order to guarantee a large enough patch in the local CRS.
         self.original_patch_size = patch_size * 3
-        kwargs['transforms'] = _Transform(K.CenterCrop(patch_size))
+        kwargs['transforms'] = _Transform(K.CenterCrop(patch_size, keepdim=True))
```

Comment:
If you have time, see if you can replace `_Transform` with `K.AugmentationSequential`. It should be possible now with `keepdim=True`.